broker-daemon/utils/migrate-store.spec.js

```diff
@@ -1,117 +1,192 @@
-const { sinon, expect } = require('test/test-helper')
+const path = require('path')
+const {
+  sinon,
+  expect,
+  rewire
+} = require('test/test-helper')
 
-const migrateStore = require('./migrate-store')
+const migrateStore = rewire(path.resolve(__dirname, './migrate-store'))
 
-describe('migrateStore', () => {
+describe.only('migrateStore', () => {
```

Comment:
You may want to add a test for processing all elements regardless of batch size. Would be nice to have a single simple test of the end-to-end functionality.

Reply:
added